@core/cms/src/external-auth-integration/auth-providers/oauth2/github/v1/github.v1.service.ts

```diff
@@ -47,8 +47,14 @@ export class GithubV1OAuth2Service implements IOAuth {
       }
 
       const tokenResponse = await exchangeCodeForToken(this.httpService, this.config, query.code);
-      Logger.log("Token response", tokenResponse);
-      res.status(HttpStatus.OK).json(tokenResponse);
+
```

Comment:
Could you describe why this change?

Reply:
the tokenResponse from GitHub is a pure string like "access_token=abcde&refresh_token=abcde&expires_in=12345", if we do `json(tokenResponse)` the result is the same string as before, not like a nicely jsonified object; the updated code will return a jsonified object like { "access_token": "abcde", "refresh_token": "abcde" }